doc/mgmt/SONiC_OC_Management_VRF_HLD.md

```diff
@@ -0,0 +1,305 @@
+           |           |  +--ro entry-type?    enumeration
+           |           +--rw interface
+           |              +--rw interface-ref
+           |                 +--rw config
+           |                 |  +--rw interface?      -> /oc-if:interfaces/interface/name
+           |                 |  +--rw subinterface?   -> /oc-if:interfaces/interface[oc-if:name=current()/../interface]/subinterfaces/subinterface/index
+           |                 +--ro state
+           |                    +--ro interface?      -> /oc-if:interfaces/interface/name
+           |                    +--ro subinterface?   -> /oc-if:interfaces/interface[oc-if:name=current()/../interface]/subinterfaces/subinterface/index
++          +--rw config
++          |  +--rw name?                       string
++          |  +--rw type?                       identityref
++          |  +--rw enabled?                    boolean
++          |  +--rw description?                string
+           |  +--rw router-id?                  yang:dotted-quad
+           |  +--rw route-distinguisher?        oc-ni-types:route-distinguisher
+           |  +--rw enabled-address-families*   identityref
+           |  +--rw mtu?                        uint16
++          +--ro state
++          |  +--ro name?                       string
++          |  +--ro type?                       identityref
++          |  +--ro enabled?                    boolean
++          |  +--ro description?                string
+           |  +--ro router-id?                  yang:dotted-quad
+           |  +--ro route-distinguisher?        oc-ni-types:route-distinguisher
+           |  +--ro enabled-address-families*   identityref
+           |  +--ro mtu?                        uint16
+           +--rw encapsulation
+
+module: sonic-mgmt-vrf
+    +--rw sonic-mgmt-vrf
+       +--rw MGMT_VRF_CONFIG
+          +--rw MGMT_VRF_CONFIG_LIST* [mgmt-vrf-name]
+             +--rw mgmt-vrf-name     string
+             +--rw mgmtVrfEnabled?   boolean 
+```
+
+### 3.6.2 CLI
+Note that the final CLI configuration and show will change based on the final data VRF configuration and show format
+
+#### 3.6.2.1 Configuration Commands
+All commands are executed in `configuration-view`:
+```
+sonic# configure terminal
+sonic(config)#
+```
+
+### Create Management VRF
+```
+sonic(config)#ip vrf
+  management               Management VRF
+  ...
+sonic(config)#ip vrf management
+sonic(config)#
+```
+
+### Delete Management VRF
+```
+sonic(config)#no ip vrf
+  management               Management VRF
+  ...
+sonic(config)#no ip vrf management
+sonic(config)#
+```
+
+#### 3.6.2.2 Show Commands
+##### Show all VRFs
+```
+sonic#show ip vrf
+VRF-Name                          Interfaces
+--------------------------------------------
+TENANT1                           Ethernet0
+
+mgmt                              eth0
+```
+
+##### Show Management VRF only
+```
+sonic#show ip vrf
+ management       Management VRF information
```

Comment:
Need to make the display "mgmt or management" consistent between show vrf & show vrf management command.

Reply:
done